src/assets/variables.scss

```diff
@@ -47,3 +47,78 @@ $opacity_full: 1;
 // menu round background hover feedback
 // good looking on dark AND white bg
 $action-background-hover: rgba(127, 127, 127, .25);
+
+// SCSS darken/lighten function override
+@function nc-darken($color, $value) {
+	@return darken($color, $value);
+}
+
+@function nc-lighten($color, $value) {
+	@return lighten($color, $value);
+}
+
+// SCSS variables
+// DEPRECATED, please use CSS4 vars
+$color-main-text: #222 !default; // Not #000 for better readability
```

Comment:
This will break theming. I think we should use CSS4 vars, like the comment indicates.

Reply:
I'll look into it, thanks!

Reply:
@ma12-co As discussed the file should not be needed in the end but all occurences of the SCSS variables should be replaced with the corresponding CSS variables.

Reply:
right!